### PR TITLE
Mod 8535 update cfda program labels

### PR DESCRIPTION
**Pull request by chasls2**
**High level description:**
Changed the CFDA label and wording to Assistance Listing

**Technical details:**
I believe all are changed. I doubled check. If one is missing please let me know

**JIRA Ticket:**
[DEV-8535](https://federal-spending-transparency.atlassian.net/browse/DEV-8535)

**Mockup:**

Author:
- [x] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

**Review by anjenkin — CHANGES_REQUESTED** [review]
I'm not seeing the fix for the following ACs:

#3, 4, 5

#7 should be “Assistance Listings (CFDA Programs)” instead of “Assistance Listings”

#8, 9

#12 - Note, backend bug is preventing testing for this one.  I see the fix in the code though.

**Comment by chasls2** [comment]
on it

**Comment by anjenkin** [comment]
I'm still seeing an issue with the following items:

#4
Screen Shot 2022-11-15 at 7 27 55 PM

#8 and 9
Screen Shot 2022-11-15 at 7 27 30 PM


#12- The text should be "Assistance Listings (CFDA Programs)".  Also I'm not sure why there's an error.  Let's plan to look at it together tomorrow.
Screen Shot 2022-11-15 at 7 29 29 PM

**Comment by chasls2** [comment]
Sorry rushed this yesterday. I didn't pull from my stash. I updated it. I believe this time all changes are done..

File: src/js/dataMapping/recipients/topCategories.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const recipientCategories = [
     'awarding_agency',
     'awarding_subagency',
     'federal_account',
-    'cfda',
+    'assistance_listings',
     'naics',
```
Comment (anjenkin) [inline src/js/dataMapping/recipients/topCategories.js:11]: The table is no longer working.  I would suspect there's an issue with the data mapping here.  I would try changing this back to cfda here and the next line that was changed to see if that fixes the problem.
Screen Shot 2022-11-17 at 3 14 17 PM

Comment (anjenkin) [reply src/js/dataMapping/recipients/topCategories.js:11]: Also the icon is missing

File: src/js/dataMapping/recipients/topCategories.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ export const recipientCategoryTitles = {
     awarding_agency: 'Awarding Agencies',
     awarding_subagency: 'Awarding Sub-Agencies',
     federal_account: 'Federal Accounts',
-    cfda: 'CFDA Programs',
+    assistance_listings: 'Assistance Listings (CFDA Programs)',
```
Comment (anjenkin) [inline src/js/dataMapping/recipients/topCategories.js:21]: I would try changing this key back to cfda and see if that fixes the issue with the data loading in the table

File: src/js/containers/covid19/helpers/covid19.jsx [hunk 3]
```diff
@@ -93,7 +93,7 @@ export const componentByCovid19Section = (publicLaw, handleExternalLinkClick) =>
         headerText: awardSpendingText,
         showInMenu: true,
         showInMainSection: true,
-        title: 'Award Spending by CFDA Program (Assistance Listing)'
+        title: 'Award Spending by Assistance Listings (CFDA Programs)'
```
[the remark this answers is not in the document]
Comment (chasls2) [reply src/js/containers/covid19/helpers/covid19.jsx]: what is the path to this page? I cant seem to find it. I think I fixed it but I cant verify it

File: src/js/dataMapping/recipients/topCategories.js (hunk 2 above)
Comment (chasls2) [reply src/js/dataMapping/recipients/topCategories.js]: fixed

File: src/js/dataMapping/recipients/topCategories.js (hunk 1 above)
Comment (chasls2) [reply src/js/dataMapping/recipients/topCategories.js]: also what is the path to this page as well. I think its fixed just need to verify

**Review by anjenkin — APPROVED** [review]

**Pull request merged**